layout/explore/explore-detail/dataset-layers/dataset-layer-card/component.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+// Styles
+import './styles.scss';
+
+function DatasetLayerCard(props) {
+  const {
+    dataset,
+    layer: { id, name, description },
+    layerGroups,
+    setMapLayerGroupActive,
+    toggleMapLayerGroup
+  } = props;
+
+  const layerGroup = layerGroups.find(lg => lg.dataset === dataset.id);
+  const layerFromRedux = layerGroup && layerGroup.layers
+    && layerGroup.layers.find(l => l.id === id);
+  const layerIsActive = layerFromRedux && layerFromRedux.active;
```

Comment:
not super urgent but for future code monkeys it might be easier if this gets resolved in a selector, so you can return from the selector `active` and `isLayerGroup` or something similar, as those are the only conditions required and they are both booleans. My preferred solution is to have functional components as dumb as possible @pablopareja 

Example selector: https://github.com/resource-watch/resource-watch/blob/develop/components/datasets/list/selectors.js